### PR TITLE
Fead/add snake gun water game

### LINKED ISSUE #4820: [New game]: Add new game Snake Water gun
### 🎮 Game Request

In this game you have to choose between snake,water and gun. Snake kills the water, water kills the gun and gun kills the snake. 

**Additional**

I have seen that someone has added this game but it is not looking good and is very simple . I have linked the video of game . If you want I can add otherwise you can close this issue .

### Point down the features

In this game winner will get 1 point. if both choose the same then both will get 1 points. in total of 10 rounds who gets the highest points will be considered as winner.

https://github.com/user-attachments/assets/df8a7d48-8f3f-4dbe-b0fd-cf489c5e0ecf

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by ANKeshri**
## PR Description 📜

In this game winner will get 1 point. if both choose the same then both will get 1 points. in total of 10 rounds who gets the highest points will be considered as winner . Please see the video attached . Its ui is lookig good . 

Fixes #4820 

<hr>
 
## Mark the task you have completed ✅

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [X] I have added screenshot for website preview in assets/images 
- [X] I have added entries for my game in main README.md
- [X] I have added README.md in my folder 
- [X] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

https://github.com/user-attachments/assets/b06116c3-2df1-42b6-87cb-3a8d3d5d6690

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @ANKeshri ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by ANKeshri** [comment]
@kunjgit please review and merge my pr

File: README.md
Comment (ishita-43) [inline README.md:1]: Please add your game entry into this file as other game and replace the ending with your game folder name

File: assets/images/image.png
Comment (ishita-43) [inline assets/images/image.png:1]: Follow contribution.md for naming convention while naming the file name of the image of your game 
eg yourgame.png

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please update your pr and make following changes

**Comment by ANKeshri** [comment]
> Please update your pr and make following changes

done the requested changes @ishita-43

**Review by ishita-43 — APPROVED** [review]
Good to go

**Review by kunjgit — APPROVED** [review]

**Comment by kunjgit** [comment]
Thank you  @ANKeshri , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**